BrowserKit/Sources/WebEngine/WKWebview/WKEngineWebView.swift

```diff
@@ -73,7 +73,7 @@ protocol WKEngineWebView: UIView {
         _ javaScript: String,
         in frame: WKFrameInfo?,
         in contentWorld: WKContentWorld,
-        completionHandler: ((Result<Any, Error>) -> Void)?
+        completionHandler: (@MainActor @Sendable (Result<Any, Error>) -> Void)?
```

Comment:
this seems to cause `MetadataFetcherHelperTests` to fail, need to investigate further